__tests__/app.test.js

```diff
@@ -164,6 +164,65 @@ describe('app', () => {
                         })
                     })
                 })
+                describe('/comments', () => {
+                    describe('GET: /api/reviews/:review_id/comments', () => {
+                        test('status 200: responds with array of comments for given review_id', () => {
+                            return request(app)
+                            .get('/api/reviews/2/comments')
+                            .expect(200)
+                            .then(({ body }) => {
+                                const { comments } = body;
+                                expect(comments).toBeInstanceOf(Array);
+                                expect(comments).toHaveLength(3)
+                                comments.forEach(comment => {
+                                    expect(comment).toEqual(
+                                        expect.objectContaining({
+                                            comment_id: expect.any(Number),
+                                            votes: expect.any(Number),
+                                            created_at: expect.any(String),
+                                            author: expect.any(String),
+                                            body: expect.any(String),
+                                            review_id: expect.any(Number)
+                                        })
+                                    )
+                                })
+                            })
+                        })
+                        test('status 200: responds with newest comments first', () => {
+                            return request(app)
+                            .get('/api/reviews/2/comments')
+                            .expect(200)
+                            .then(({ body }) => {
+                                const { comments } = body;
+                                expect(comments).toBeSortedBy('created_at', { descending: true });
+                            })
+                        })
```

Comment:
You could add a test to check what happens when we request an article with no comments - I'd expect a 200 status and an empty array to be returned